### PR TITLE
[ADVAPP-389]: Update label for Interactions > Campaigns (cleanup)

### PR DESCRIPTION
**Pull request by dgoetzit**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-389

### Technical Description

> This PR cleans up and removes hanging references to the Interaction Campaign, which was removed in favor of Interaction Initiatives

### Any deployment steps required?

> No

_______________________________________________

### CODE REVIEW

**Comment by Orrison** [comment]
@dgoetzit looks like there is a merge conflict on this.

Also, is is possible to make it into a draft since we should not merge this yet? Finally, if you can do me a favor and make sure there is a ticket for this at the top of the backlog.

File: app-modules/interaction/graphql/interaction.graphql
Comment (dgoetzit) [reply app-modules/interaction/graphql/interaction.graphql:1]: This has been restored.

File: app-modules/interaction/graphql/interaction.graphql
Comment (Orrison) [inline app-modules/interaction/graphql/interaction.graphql:40]: I think this probably should be called `initiative` now, right?

**Review by Orrison — CHANGES_REQUESTED** [review]

**Pull request merged**
